README.rst

```diff
@@ -32,21 +32,43 @@ Javascript. With Batavia, you can run Python bytecode in your browser.
 It honors Python 3.4.4+ syntax and conventions, and let's you
 reference objects and classes defined natively in JavaScript.
 
-Tutorial
---------
+Quick Start
+---------------
 
-To take Batavia for a spin, run through the `Getting Started guide`_.
+(For a more detailed setup, please see the `Getting Started tutorial <https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html>`_
 
-Then have some fun with `the first tutorial`_, and try out running Python in
-your browser.
+Batavia requires a Python 3.4 installation, a copy of `Ouroboros <https://github.com/pybee/ouroboros>`_, and a virtualenv to run it all in. 
 
-.. _Getting Started guide: https://batavia.readthedocs.io/en/latest/intro/getting-started.html
-.. _the first tutorial: https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html
+Clone the code repos:: 
+
+   $ mkdir pybee
+   $ cd pybee
+   $ git clone https://github.com/pybee/batavia
+   $ git clone https://github.com/pybee/ouroboros
+
+Setup a virtualenv (for other environments, see `Getting Started <https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html>`_)::
+
+   $ virtualenv venv
+   $ . venv/bin/activate
+   $ cd batavia
+   $ pip install -e .
+
+And finally, run the :code:`make` script to generate the combined Batavia JavaScript files::
+
+   $ make
+
+Python in the Browser
+----------------------
+
+Once you've setup your environment, you can have some fun with `running Python in your browser <https://batavia.readthedocs.io/en/latest/tutorials/tutorial-1.html>`_
```

Comment:
Should this be a separate heading, or just run into a couple more quick start commands - i.e., 

    $ cd testserver
    $ pip install -r requirements.txt
    $ ./manage.py runserver

then open a web browser at localhost:8080.